packages/website/plugins/generated-rule-docs.ts

```diff
@@ -65,34 +67,28 @@ export const generatedRuleDocs: Plugin = () => {
       type: 'blockquote',
     } as mdast.Blockquote);
 
-    // 3. Add a rule attributes list...
-    const attributesH2Index =
-      // ...before the first h2, if it exists...
-      parent.children.findIndex(
-        child => nodeIsHeading(child) && child.depth === 2,
-      ) ??
-      // ...or at the very end, if not.
-      parent.children.length;
-
-    // The actual content will be injected on client side.
+    // 3. Add a rule attributes list before the first h2.
+    const attributesH2Index = root.children.findIndex(
+      child => nodeIsHeading(child) && child.depth === 2,
+    );
```

Comment:
It was pretty silly to use `??`, but we probably still need to guard against `-1`?

Reply:
🤷 it's been working ever since! I guess all our docs have h2s.

Reply:
That's true... Just a bit of defensiveness. I guess it doesn't really matter in that case.